testsuite/cluster-test/src/cluster_builder.rs

```diff
@@ -48,8 +48,8 @@ impl ClusterBuilderParams {
 }
 
 pub struct ClusterBuilder {
-    current_tag: String,
-    cluster_swarm: ClusterSwarmKube,
+    pub current_tag: String,
+    pub cluster_swarm: ClusterSwarmKube,
```

Comment:
This does not need to be `pub` anymore rigth?

Reply:
`current_tag` needs to be `pub` because we're editing it in place in `compatibility_test`, but yea not `cluster_swarm`